python/lsst/metaserv/schemaToMeta.py

```diff
@@ -47,7 +47,7 @@ class SchemaToMeta(object):
 
     _tableStart = re.compile(r'CREATE TABLE (\w+)*')
```

Comment:
I don't think you want the `*` here.  That would allow a line with no table name.